src/beamlime/applications/daemons.py

```diff
@@ -61,3 +74,36 @@ async def run(self) -> None:
         )
 
         self.info("Data streaming finished...")
+
+
+class FakeListener(DaemonInterface):
```

Comment:
I think we should replace https://github.com/scipp/beamlime/blob/8958ea820d994e60941c56cd178797443f3db507/src/beamlime/executables/prototypes.py#L29 so it uses this instead. The `DataStreamSimulator` should be removed. What do you think @YooSunYoung?

Reply:
Me and @YooSunYoung talked about this yesterday and then we came to the conclusion that it's better to remove it in the PR where fake events are implemented